ipatests/test_integration/test_passkey.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (C) 2020  FreeIPA Contributors see COPYING for license
+#
+
+"""
+Tests to verify Passkey Feature.
+"""
+
+from __future__ import absolute_import
+
+import os
+import pytest
+import textwrap
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.pytest_ipa.integration import tasks
+
+
+def setup_files_on_host(host, file):
+    dest = "test_integration/data/{0}".format(file)
+    here = os.path.dirname(os.path.realpath('__file__'))
+    with open(os.path.join(here, dest), 'rb') as f:
+        contents = f.read()
+    path = "/opt/{0}".format(file)
+    host.put_file_contents(path, contents)
+    return path
+
+
+class TestPasskey(IntegrationTest):
+    """ Tests for passkey feature.
+
+    This test class covers the tests for Passkey feature using umockdev
+    utility. With umockdev utility required files are generated with
+    passkey input using umockdev-record and stored in
+    ipatests/test_integration/data/passkey_* and used for umockdev-run.
+
```

Comment:
Hi @amore17 
can you expand a bit the description with detailed explanations how to generate the files? For instance:
- dnf install -y umockdev
- insert the passkey
- run "umockdev-record --ioctl /dev/hidraw1=passkey_test2hidraw1.ioctl --script /dev/hidraw1=passkey_test2hidraw1.script -- ipa user-add-passkey ..."
It would be nice to have the exact recipe as everyone is not familiar with umockdev.

Otherwise LGTM, you can remove the temp commit.